### PR TITLE
fix: load json within context

### PR DESCRIPTION
**Pull request by estellerambier**
This is adding context to the loading of the BigQuery credential JSON file.

### CODE REVIEW

**Comment by estellerambier** [comment]
Hey @MaxHalford, I wanted to try out your library, and I wasn't able to load the credentials without this change. It's quite enigmatic to me. Would you have an idea why?

File: lea/clients/__init__.py [hunk 1]
```diff
@@ -20,9 +20,13 @@ def make_client(production: bool, wap_mode=False):
         scopes = scopes_str.split(",")
         scopes = [scope.strip() for scope in scopes]
 
+        service_account_info_path = os.environ["LEA_BQ_SERVICE_ACCOUNT"]
+        with open(service_account_info_path, "r") as file:
+            service_account_info = json.load(file)
```
Comment (MaxHalford) [inline lea/clients/__init__.py:25]:
```suggestion
        with open(service_account_info_path, "r") as f:
            service_account_info = json.load(f)
```

**Review by MaxHalford — APPROVED** [review]
Thanks for the PR!

Indeed it's weird why the current code doesn't work for you. I'm not exactly sure. However, I've tested your change, and it works fine for me, so let's go with it! I just added a nitpick comment :)

**Comment by estellerambier** [comment]
I let you merge, I don't have the right

**Comment by MaxHalford** [comment]
Done! I'll loosen permissions for next time

File: lea/clients/__init__.py [hunk 2]
```diff
@@ -20,9 +20,13 @@ def make_client(production: bool, wap_mode=False):
         scopes = scopes_str.split(",")
         scopes = [scope.strip() for scope in scopes]
 
+        service_account_info_path = os.environ["LEA_BQ_SERVICE_ACCOUNT"]
```
Comment (estellerambier) [inline lea/clients/__init__.py:23]: I hope we didn't go too quickly. I see now that in the doc the `LEA_BQ_SERVICE_ACCOUNT` is not expecting a path but a key-value format (a JSON dump). This is why it wasn't working for me, as I assigned the path of the `.json` to the variable

**Pull request merged**